src/site/content/en/blog/catalina-system-ui/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: More variable font options for the macOS `system-ui` font in Chromium 83
+subhead: Catalina brings a new united variable system font to macOS 
+authors:
+  - adamargyle
+  - drott
+description: Chromium 80 had a system-ui font weight regression on macOS. The reason it broke, and the new super powers post-resolution, are worth the wait in Chromium 83.
+date: 2020-05-21
+updated: 2020-05-21
+hero: hero.jpg
+thumbnail: thumb.png
+alt: A bright pink and purple gradient with "macOS Catalina system-ui" going from thin to think from left to right, demonstrating some of the new variation settings
+tags:
+  - blog
+  - css
+  - fonts
+---
+
+The ['system-ui' section](https://drafts.csswg.org/css-fonts-4/#system-ui-def) of the CSS Fonts Module Level 4 spec defines a `system-ui` font keyword that allows developers to use the built-in, turbo-optimized, localized, mega-high-quality, no-download-needed, default operating system font font right in their sites and apps. 
+
+```css
+body {
+  font-family: system-ui;
+}
+```
+
+This typography choice is akin to saying:
+
+{% Blockquote 'browser' %}
+use the native system font for the current locale of this user
+{% endBlockquote %}
+
+On macOS, the `system-ui` font is San Francisco, a font that a design team vetted, tested, and… recently upgraded! First we'll cover the [new exciting variable font features in Catalina](#new-powers), then we'll cover a couple of [bugs and how Chromium engineers resolved them](#regression).
+
+This post assumes that you're already familiar with variable fonts. If not, check out [Introduction to variable fonts on the web](/variable-fonts/) and the video below.
+
+{% YouTube 'B42rUMdcB7c' %}
+
+### Browser Compatibility
+At the time of writing, `system-ui` has support from Chromium (since 56), Edge (since 79), Safari (since 11), and from Firefox (since 43) but with the `-apple-system` keyword. See Can I use variable fonts? for updates.
```

Comment:
```suggestion
At the time of writing, `system-ui` has support from Chromium (since 56), Edge (since 79), Safari (since 11), and from Firefox (since 43) but with the `-apple-system` keyword. See [Can I use variable fonts?](https://caniuse.com/#feat=variable-fonts) for updates.
```